test/lb-ng.test.js

```diff
@@ -59,6 +60,18 @@ describe('lb-ng', function() {
       });
   });
 
+  it('supports async booting apps', function() {
+    this.timeout(3000);
```

Comment:
This shouldn't be needed once the app uses `process.nextTick` instead of a long timeout.